### ISSUE TITLE
Bedrock Knowledge Base Search Filter "in" broken

### ISSUE THREAD

**Issue by manuwaik** [langchain-ai/langchain-aws#169]
https://github.com/langchain-ai/langchain-aws/blame/d89fcf80c61174c21c88ef8d1cfbf3a55c7d84c5/libs/aws/langchain_aws/retrievers/bedrock.py#L24

I get error:
```
Parameter validation failed:
Unknown parameter in retrievalConfiguration.vectorSearchConfiguration.filter: "in_", must be one of: andAll, equals, greaterThan, greaterThanOrEquals, in, lessThan, lessThanOrEquals, listContains, notEquals, notIn, orAll, startsWith, stringContains
```

**Comment by isahers1** [comment]
Could you show a full code snippet that is causing the error so I can help debug?

**Comment by manuwaik** [comment]
```
# Retrieve chunks from relevant documents
    retriever = AmazonKnowledgeBasesRetriever(
        knowledge_base_id="XXXXXXXX",
        retrieval_config={"vectorSearchConfiguration": 
                            {
                                "numberOfResults": 5,
                                "filter": {
                                    "in": {
                                        "key": "tag",
                                        "value": ["document1", "document2"]
                                        }
                                    }
                            }
                         },
    )

    docs = retriever.invoke(query)
```
The metadata with key `tag` exists on the KB. 
I have also tried using `in_`.

**Comment by karoaper** [comment]
The problem is here: https://github.com/langchain-ai/langchain-aws/blob/main/libs/aws/langchain_aws/retrievers/bedrock.py#L165.  The call to model_dump should include "by_alias=True".

**Comment by renjiexu-amzn** [comment]
Oh, karoaper beated me on commenting the root cause. Would you help merge my change in #233  and release a new minor version with the fix? Do you have an ETA on the release so that I could coordinate the work blocked by this one?

**Comment by 3coins** [comment]
Fixed in v0.2.3.
